ownCloud Share Extension/ShareViewController.swift

```diff
@@ -96,6 +97,18 @@ class ShareViewController: MoreStaticTableViewController {
 			didAppearInitial = true
 		}
 	}
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillAppear(animated)
```

Comment:
`viewWill_Disappear_()` calls `super.viewWill_Appear_()` here. That should probably be `super.viewWill_Disappear_()`